zingolib/src/lightclient.rs

```diff
@@ -2055,6 +2057,27 @@ impl LightClient {
 
         Ok(())
     }
+
+    /// Some LightClients have a data dir in state. Mobile versions instead rely on a buffer and will return an error if this function is called.
```

Comment:
I see the exact same doc-comment twice.

Here's an an example of a doc-comment in `librustzcash`:

https://github.com/zcash/librustzcash/blob/29c676ff125406e4b9f20fd046d103f42aa85e31/zcash_primitives/src/transaction/fees/zip317.rs#L42

Run `cargo doc --open` to see how the doc-comment is rendered.